test/runner/requirements/ansible-test.txt

```diff
@@ -1 +1,2 @@
 argparse ; python_version < '2.7'
+toml >= 0.10
```

Comment:
Requirements for individual integration tests should be installed during the test.

Reply:
Sorry, would you tell me how to add requirements for a test case?

Reply:
I mean where I would to add dependencies.